### ISSUE TITLE
Point HACKING to Stash documentation

### ISSUE THREAD

**Issue by elextr** [geany/geany#919]
To encourage making "proper" prefs that users can find, instead of "various" prefs that nobody sees, the HACKING should point to the stash docs in the plugin API.

Will make PR "soon" if nobody beats me to it.

**Comment by codebrainz** [comment]
IMO, it would be more useful to list the steps to add a preference in `HACKING` file, like it has for adding a filetype.

**Comment by elextr** [comment]
Yep, and for the details still needs to point to the API as well.

**Comment by kugel-** [comment]
Waiting for the PR :-)

**Comment by elextr** [comment]
Its still "soon" by my schedule :)
